server/app.js

```diff
@@ -1,8 +1,9 @@
 import http from 'node:http'
```

Comment:
How local development will looks like with these changes?

We need a way to rebuild assets automatically.

Reply:
Now `pnpm start` on the server calls `pnpm -r build && node app.js`.

Reply:
How we will rebuild frontend files on changes?

I think we need some `watch` script which will run Vite with watch argument (if it has it).

Reply:
We can run `vite build --watch` :)

Reply:
Sure, can you add it to `pnpm start` in the core?

Reply:
![image](https://user-images.githubusercontent.com/22644149/183915659-334da6b4-0ba8-4fe6-96c5-beb50704861b.png)

Reply:
> Sure, can you add it to pnpm start in the core?

Shouldn't this be done in the server code?

Reply:
https://github.com/browserslist/browserl.ist/pull/319#issuecomment-1210758841